core/src/main/java/nu/yona/server/subscriptions/service/UserPrivateDataBaseDto.java

```diff
@@ -69,4 +69,10 @@ public String getNickname()
 
 	@JsonIgnore
 	public abstract boolean isFetchable();
+
+	public boolean isChanged(UserPrivateDataBaseDto original)
```

Comment:
I find it confusing that those `isChanged` methods only have a subset of the properties included. It is named as a general-purpose method, but actually does something specific. For example, why would the `userPhotoId` or even the `goals` not be included here? That requires knowledge on how the controllers and services use this class, which is not nice to put in this class itself.

Reply:
I understand what you say. What about moving these comparison operators to the service that uses them (e.g. the ``UserUpdateService`` and name them ``containsPropertyUpdate``?

Reply:
Sounds better to me!